packages/backend/src/services/OAuthService/OAuthService.ts

```diff
@@ -80,47 +80,4 @@ export class OAuthService extends BaseService {
         if (refreshToken) return this.oauthModel.revokeToken(refreshToken);
         return false;
     }
-
-    public async authenticateWithOauthToken(
```

Comment:
this custom method is no longer needed, now we use proper oauthserver.authenticate using the app token int he header.